Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -35,6 +35,43 @@ public object Get()
         }
 
         ////TODO: add automatic conversions
+        /// <summary>
+        /// Read the value of the action.
+        /// </summary>
+        /// <returns>The current value from the action cast to the specified type.</returns>
+        /// <typeparam name="TValue">Type of value to read. This must correspond to the
+        /// expected by either <see cref="control"/> or, if it is a composite, by the
+        /// <see cref="InputBindingComposite"/> in use.
+        /// The type depends on what type of controls the action is bound to. Common types are float and Vector2</typeparam>
```

Comment:
I think float should be code formatted e.g. <c>float</c>

and <see cref="UnityEngine.Vector2" />

Reply:
Good idea - doing it

(after editing above so I can see it ;) )